zetaclient/common/constant.go

```diff
@@ -1,6 +1,9 @@
 package common
 
 const (
+	// DefaultGasPriceMultiplier is the default gas price multiplier for all chains
+	DefaultGasPriceMultiplier = 1.0
```

Comment:
Why needed?

Reply:
It is now used by `PostVoteGasPrice`. EVM chain uses `1.2`, bitcoin uses `2.0`.  What's question?